creating_images/metadata.adoc

```diff
@@ -115,7 +115,7 @@ LABEL io.openshift.non-scalable     true
 |This label suggests how much resources the container image might need in order to
 work properly. The UI might warn the user that deploying this container image may
 exceed their user quota.  The values must be compatible with
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/resources.md#resource-quantities[Kubernetes
+https://kubernetes.io/docs/concepts/policy/resource-quotas/[Kubernetes
```

Comment:
@iashok22 The earlier link here was taking about allowed quantities. For example 10Gi for storage, 3 for cpu. Resource Quotas is a different thing. Can you please check and updated this link?

Reply:
Please fell free to remove the link if you cannot find anything matching.